code/modules/mob/living/silicon/robot/robot_model.dm

```diff
@@ -45,6 +45,8 @@
 	var/magpulsing = FALSE
 	/// Clean trash under the borg
 	var/clean_on_move = FALSE
+	/// Whether the cyborg has extra armor slowing it down and reducing incoming damage
+	var/cyborg_armor = FALSE
```

Comment:
Is this good implementation? Shouldn't an action of putting the armor just change the values? What's the purpose of tracking it here? And anyway if armor is extra addition, can't you track it in something like modules?

Reply:
It matches the current implementation of tracking other module specific variables which are applied during module application. I can look at improving it, but overhauling how cyborgs transform wasn't really on the table for this PR.